File: ApiSuvesaPos/Negocio/Modelo/inventario/familia/familias.vb
Namespace Modelo
    Public Class familias
        Public Property Codigo As Integer
        Public Property Descripcion As String
        Public Property Observaciones As String
        Public Property CuentaGra As String
        Public Property DescripcionGra As String
        Public Property CuentaExe As String
        Public Property DescripcionExe As String
        Public Property CuentaCosto As String
        Public Property DescripcionCosto As String
        Public Property SubFamilias As List(Of subFamilias)
    End Class

End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/familia/familias.cs
namespace Modelo
{
    public class familias
    {
        public int Codigo { get; set; }
        public string Descripcion { get; set; }
        public string Observaciones { get; set; }
        public string CuentaGra { get; set; }
        public string DescripcionGra { get; set; }
        public string CuentaExe { get; set; }
        public string DescripcionExe { get; set; }
        public string CuentaCosto { get; set; }
        public string DescripcionCosto { get; set; }
        public List<subFamilias> SubFamilias { get; set; }
    }
}
